### ISSUE TITLE
Blockout multiple selectors - how does it work?

### ISSUE THREAD

**Issue by Gjodim** [tacoss/testcafe-blink-diff#38]
Hello. I am in  a bit of a pickle and perhaps am doing something wrong.

I am trying to blockout **multiple** selectors on a page in a single test.

What i try is the following:

`blockOut: [Selector('.eventlist'), Selector('.eventlist')]`

but it does not seem to work.
My question / issue is - is there any other syntax i should use for blocking multiple selectors?

**Comment by pateketrueke** [comment]
Hi, according [the test](https://github.com/tacoss/testcafe-blink-diff/blob/45ad90a99f3e0418b883ebab2d1f8d953f086cfa/e2e/cases/main.test.js#L34-L52) we did to cover this feature that is the correct syntax.

Just to confirm, during your tests a file `blockOut.json` should be written in the same place where the base.png screenshots are stored, can you confirm it is created as well?

That should contain all the coordinates on the DOM that will be blocked-out, if the file is present and stilll is not working, probably is due the `blink-diff` dependency, may be we just need to upgrade, etc.

**Comment by Gjodim** [comment]
Yes, a blockout.json is created with seemingly the correct coordinates.
I will try to take the snapshots again with some other selector combination which should yield the same result.

If this doesn't work is there something else (workaround or something) you recommend i try?

**Comment by Gjodim** [comment]
Checked again but it still does the comparison - blockout.json is generated but it doesn't seem to consider the blockedOut areas.

**Comment by pateketrueke** [comment]
> it doesn't seem to consider the blockedOut areas.

Well, in the actual tests we did for it, the resulting images will no have such areas deleted, or filled, or anything. They'll remain as  the original screenshot, but they're just ignored during image comparison.

Did you ran a quick test to confirm that?

**Comment by Gjodim** [comment]
Yes. Here is an image of the results
<img width="1709" alt="image" src="https://user-images.githubusercontent.com/46923914/186176865-83de5a61-365c-4330-8f66-d1fc5284707c.png">


And here is from the folder structure of the failing images - base and actual + blockOut.json
<img width="540" alt="image" src="https://user-images.githubusercontent.com/46923914/186178025-1c1b9971-3445-4e9a-90f3-f6d56664f11f.png">

**Comment by pateketrueke** [comment]
Open the diff of the images that did no pass, the actual blocked-out areas should remain, without the red indicator.

**Comment by Gjodim** [comment]
**base**
<img width="1118" alt="image" src="https://user-images.githubusercontent.com/46923914/186241933-0fbd6644-d9ce-4f1b-ab33-39471b2eef76.png">

**actual**
<img width="1101" alt="image" src="https://user-images.githubusercontent.com/46923914/186241974-b4a366f4-0cf6-4c55-b44a-496799096cdf.png">


**blockOut.json**
[{"left":0,"right":1200,"top":160,"bottom":729,"width":1200,"height":569}]

**Comment by pateketrueke** [comment]
I mean, when you get the dashboard, click on the "Open diff" action, to see the actual difference.

**Comment by Gjodim** [comment]
Sorry i am not sure i understand.
Perhaps this:
<img width="962" alt="image" src="https://user-images.githubusercontent.com/46923914/186440536-8261f571-afd3-4312-9521-210d8d16f8c3.png">

**Comment by pateketrueke** [comment]
Yeah, that screen! And sadly is not working... I've been testing the blockOut feature (blink-diff provides it) but without luck, I think we should start consider alternatives for the visual diffing library. 🤔

**Comment by pateketrueke** [comment]
Update: I replaced `blink-diff` with `native-image-diff`and `node-libpng` &mdash; It seems to be working great, but I am reworking the UI a bit due changes on the generated images.

> Said that, I think `testcafe-blink-diff` is not longer an accurate name... we should rename the module to a be more neutral? 💭

**Comment by Gjodim** [comment]
Hello @pateketrueke . Should i install some other package if this is fixed, or?

**Comment by pateketrueke** [comment]
Justa upgrade, from the user side nothing else is needed.
